### DISCUSSION TITLE
when importing a dataset with LENA recordings: splitting recordings from a session

### DISCUSSION THREAD

**Discussion by alecristia** [LAAC-LSCP/ChildProject#311] [Q&A]
LENA outputs a single wav for all recordings in a given session. We have decided that in ChildProject, the raw waves may be like that, but the converted version and all annotations that refer to them should be split.

When and how does one split raw wavs from a session? Sorry if I missed instructions for this somewhere!

**Comment by lucasgautheron** [comment]
> LENA outputs a single wav for all recordings in a given session. We have decided that in ChildProject, the raw waves may be like that, but the converted version and all annotations that refer to them should be split.
> 
> When and how does one split raw wavs from a session? Sorry if I missed instructions for this somewhere!

The decision to split or not should be fully consistent. In El1k we decided to split the audio, but since the audio is not included except for Lyon, it only affected the metadata and annotations.

The code to split the audio has not been included into the EL1000 package, but I can do that tomorrow morning. It not very complicated (here is the code for Lyon for instance; most of the complexity comes from the fact that the audio filenames on HomeBank did not match the metadata: https://gin.g-node.org/EL1000/lyon/src/main/scripts/split.py

**Reply by alecristia** [reply]
ah actually, this is for Lyon. I'm pushing everything (including its, upl, wav) right now, but overall only 26% done after 3h of transfer, so best guess is that it'll be uploaded by tomorrow's work day

**Reply by lucasgautheron** [reply]
Is it still uploading? Can't see the data on GIN; can you push again if the transfer got interrupted?
Thanks!

**Comment by lucasgautheron** [comment]
Alright!

Le lun. 18 oct. 2021 à 14:01, Alex Cristia ***@***.***> a
écrit :

> ah actually, this is for Lyon. I'm pushing everything (including its, upl,
> wav) right now, but overall only 26% done after 3h of transfer, so best
> guess is that it'll be uploaded by tomorrow's work day
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/LAAC-LSCP/ChildProject/discussions/311#discussioncomment-1495043>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABI4QHQILEGOXUE7NMZAPYLUHQEA5ANCNFSM5GGEDILQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Lucas Gautheron
06 79 23 86 47

Laboratoire de Sciences Cognitives et Psycholinguistique
Bureau 414 - 29 rue d'Ulm
75005 Paris

**Comment by alecristia** [comment]
yes!

On Tue 19 Oct 2021 at 09:09, Lucas Gautheron ***@***.***>
wrote:

> Is it still uploading? Can't see the data on GIN; can you push again if
> the transfer got interrupted?
> Thanks!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/LAAC-LSCP/ChildProject/discussions/311#discussioncomment-1500354>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABY6PHOEMJZZN27FIPJQ2O3UHUKRRANCNFSM5GGEDILQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>
-- 

---------------------------------------------------------------
Alex (Alejandrina) Cristia
Researcher, CNRS
Laboratoire de Sciences Cognitives et Psycholinguistique
29, rue d'Ulm, 75005, Paris, FRANCE
My site: www.acristia.org
---------------------------------------------------------------
